### ISSUE TITLE
exporting yolo dataset is failed.

### ISSUE THREAD

**Issue by siegkeron** [snuailab/waffle_hub#141]
### Search before asking

- [X] I have searched the [issues](https://github.com///issues) and found no similar bug report.

### Bug

When I tried to load dataset as
```
dataset = Dataset.load(
    "st_cls",
    root_dir="hubs"
)
dataset_dir = dataset.export(DataType.YOLO)
```

error msg occur as

```
FileNotFoundError: [Errno 2] No such file or directory: 'hubs/st_cls/exports/YOLO'
```
I don't know this is intended or not.
But in my opinion, 'exports/YOLO' path should be created when I execute "dataset.export(DataType.YOLO)"

**Comment by ZeroAct** [comment]
For clarifying the issue, is the `root_dir` of your dataset is `hubs`?

**Comment by siegkeron** [comment]
I found it caused by function 

dataset/dataset.py line 993-1006 def get_split_ids(self): 

and error occur with 
```
unlabeled_ids: list[int] = io.load_json(self.unlabeled_set_file)
```
becuase "unlabeled sets" json file is not exist.

I make dataset by
```
dataset = Dataset.from_yolo(
    name='st_cls',
    task=TaskType.CLASSIFICATION,
    yaml_path='st_dataset/data.yaml',
    root_dir='hubs'
)
```
with data.yaml as
```
train: train
val: val
test: val
path: st_dataset
nc: 2
names:
  0: '0'
  1: '1'
```

**Comment by ZeroAct** [comment]
Oh, I see.
Thank you for reporting the bug.
We will fix it in version 0.2.1 today.
